### PR TITLE
Teacher admin group7

### PR DESCRIPTION
**Pull request by ryan-hafen-525**
Our team has updated the teacher admin controller, EF repo, and views to include the models. Styling isn't done yet.

### CODE REVIEW

**Comment by ryan-hafen-525** [comment]
There were no build errors; however, due to the way the database was set up, the total functionality of the views interacting with the models may not work. Need to talk with the database team to get some of the models changed.

**Comment by ryan-hafen-525** [comment]
I added in a new field in the model just as a heads up. It was necessary in order to display the view correctly. Not sure how we'd want to go about fixing that.

**Comment by ch843** [comment]
I had to move the views from you admin_ folders to TeacherAdmin so the controller could recognize them based on Coding by Convention. Just an FYI!

File: Controllers/TeacherAdminController.cs [hunk 1]
```diff
@@ -26,5 +26,102 @@ public IActionResult JudgeView()
...
+            return View("rubric_Index");
+        }
+
+        // Methods to update a rubric item
+        public IActionResult Edit_Rubric_Item(int id)
+        {
+            var rubricItem = _repo.Rubric_Items
+                .Single(x => x.rubric_item_id == id);
+
+            return View("rubric_edit_scores", rubricItem);
```
Comment (ch843) [inline Controllers/TeacherAdminController.cs]: When I pull this up in Rider, it is throwing an error. Is this the right way to pass rubricItem to a view?

File: Controllers/TeacherAdminController.cs [hunk 2]
```diff
@@ -26,5 +26,102 @@ public IActionResult JudgeView()
...
+            _repo.Team7Method3(rubricItem);
+
+            return RedirectToAction("View_Rubric_Category");
+        }
+
+        // Methods to add a new rubric item
+        [HttpGet]
+        public IActionResult Add_Rubric_Item()
+        {
+            return View("rubric_edit_scores", new Rubric_Item());
```
Comment (ch843) [inline Controllers/TeacherAdminController.cs:118]: I am also getting an error on this line with new Rubric_Item()

**Review by ch843 — CHANGES_REQUESTED** [review]
I left two comments in TeacherAdminController that need to be looked at, but other than that great job team!!

**Comment by ch843** [comment]
Merged into rubric views. Styling still needs to be fixed but we will do that later.

**Pull request merged**